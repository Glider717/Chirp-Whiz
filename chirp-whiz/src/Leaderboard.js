import React, { useState, useEffect } from 'react';
import 'bootstrap/dist/css/bootstrap.min.css';
import Table from 'react-bootstrap/Table';

function compare(a, b) {
    const scoreA = a.score;
    const scoreB = b.score;

    let comparison = 0;
    if (scoreA > scoreB) {
        comparison = 1;
    } else if (scoreA < scoreB) {
        comparison = -1;
    }

    return comparison;
}