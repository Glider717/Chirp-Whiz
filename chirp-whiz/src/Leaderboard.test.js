import React from 'react';
import Leaderboard from './Leaderboard';
import '@testing-library/jest-dom/extend-expect'
import { render, fireEvent, waitForElement } from '@testing-library/react'

