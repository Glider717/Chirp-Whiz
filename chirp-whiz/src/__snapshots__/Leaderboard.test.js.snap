// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Leaderboard renders 1`] = `undefined`;
